src/WebUI/Pods/PodContentReader.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    Licensed under the MIT license.
+*/
+
+import { BaseComponent, css } from "@uifabric/utilities";
+import { CardContent, CustomCard } from "azure-devops-ui/Card";
+import * as React from "react";
+import { IVssComponentProperties } from "../Types";
+
+// basic editor functionality
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
+// languages supported in editor
+import "monaco-editor/esm/vs/basic-languages/yaml/yaml.contribution";
+// additional functionality of editor
+import "monaco-editor/esm/vs/editor/browser/controller/coreCommands";
+import "monaco-editor/esm/vs/editor/contrib/find/findController";
+import "monaco-editor/esm/vs/editor/contrib/message/messageController";

```

Comment:
messageController helps us to show a message when user tries to edit YAML
![image](https://user-images.githubusercontent.com/26214977/55156877-09e2cb80-5181-11e9-93c4-cb9e8574ea72.png)